tests/common/plugins/test_completeness/README.md

```diff
@@ -3,20 +3,23 @@ CompletenessLevel marker enables testcases to be executed in different meaningfu
 Each level is a representation of a scope of execution of a testcase. This document describes the usage of CompletenessLevel marker.
 Identified meaningful levels (in increasing order) -
         
-        Debug
+        debug
         
-        Basic
+        basic
         
-        Confident
+        confident
         
-        Thorough
+        thorough
+
+An unordered level `diagnose` is also supported. If `diagnose` level is specified and a test doesn’t support it, then it shall fall back to `basic` mode. This level doesn’t have any implication of above or below any existing level. Furthermore, this level is not the same as `debug` level, which is intended to take a sample test to debug test code. `diagnose` level is to be used to diagnose sonic feature/behavior.
```

Comment:
> `diagnose` level is used to diagnose sonic feature/behavior.

Can you clarify this some more? I think it is still a little confusing what the difference is between this new level and the existing levels. Maybe an example use-case would help?

Reply:
Improved the description and described what the level is ideally meant for.